code/modules/games/cards.dm

```diff
@@ -415,9 +415,6 @@
 	var/obj/item/cardhand/C = target
 	if(length(C.cards) <= 1)
 		return FALSE
-	var/mob/living/carbon/human/O = owner
-	if(O.l_hand && O.r_hand)
```

Comment:
I _think_ what this is trying to do is make the action unuseable if you have something in both your hands, right?

Reply:
it doesn't

Reply:
can you check to make sure that this doesn't break the `remove_card` action. I believe this is intended to make sure that the user doesn't just remove the card onto the ground if they have nothing in their hands.

Reply:
> can you check to make sure that this doesn't break the `remove_card` action. I believe this is intended to make sure that the user doesn't just remove the card onto the ground if they have nothing in their hands.

fixed

Reply:
can you check to make sure that if the users hands are full, that they don't just drop the card on the ground?